nlb/nlb.go

```diff
@@ -342,54 +342,53 @@ func generateTargetDescriptionFromTargetType(targetType, instanceID, privateIP s
 
 // Stop removes this instance from all the front end NLBs
 func (e *nlb) Stop() error {
-	var failed = false
-	for _, elb := range e.elbs {
-		log.Infof("Deregistering instance %s with nlb %s", e.privateIPAddress, elb.Name)
-		err := deregisterFromLoadBalancer(e, elb)
-		if err != nil {
-			log.Warnf("unable to deregister instance %s (%s) with nlb %s: %v", e.instanceID, e.privateIPAddress, elb.Name, err)
-			failed = true
+	failedCount := 0
+	successCount := 0
```

Comment:
These could just be booleans I think.

Reply:
Yeh sure. My initial idea was to use the number which failed for some elaborate logging but now that I did not do that, I can change those to booleans